### PR TITLE
Make the output of incremental compatible with black formatting tool

### PR DESCRIPTION
**Pull request by rodrigc**
Fixes https://github.com/twisted/incremental/issues/55

### CODE REVIEW

**Comment by wsanchez** [comment]
This PR does a lot more than the header; making incremental use black itself is not related to making incremental's output compatible with black.

Given that making incremental's output compatible with black is important to clients, I think it would be important to do just that in a PR, so that we can document that change independently.

File: .travis.yml [hunk 1]
```diff
@@ -11,13 +11,11 @@ env:
...
-    - python: 3.5
-      env: TOX_ENV=py35-flake8
-    - python: 2.7
-      env: TOX_ENV=py27-flake8
-    - python: 2.7
+    - python: 3.6
+      env: TOX_ENV=tests
+    - python: 3.6
+      env: TOX_ENV=black
+    - python: 3.6
```
Comment (wsanchez) [inline .travis.yml:18]: Seems like the first should run on 3.6, 3.7, 3.8 and the rest on 3.8, no?

**Comment by rodrigc** [comment]
> This PR does a lot more than the header; making incremental use black itself is not related to making incremental's output compatible with black.
> 
> Given that making incremental's output compatible with black is important to clients, I think it would be important to do just that in a PR, so that we can document that change independently.

OK, I removed all parts of the PR which formats the incremental code itself with black.
This PR just contains the changes which make the output black compatible

File: .travis.yml (hunk 1 above)
Comment (rodrigc) [reply .travis.yml]: I've moved this to https://github.com/twisted/incremental/pull/64

**Comment by rodrigc** [comment]
I've moved all patches which reformats the incremental tree with black to https://github.com/twisted/incremental/pull/64

**Pull request merged**